src/net.h

```diff
@@ -294,6 +295,8 @@ class CConnman
     void SetBestHeight(int height);
     int GetBestHeight() const;
 
+    /** Get a unique deterministic randomizer. */
+    CSipHasher GetDeterministicRandomizer(uint64_t id);
```

Comment:
Is this something that belongs on CConnman? Conceptually it seems a bit odd to have randomization on the network manager.
E.g. Could there be reasons to have deterministic randomizers that don't have to do with networking? Not sure. Maybe for randomizing data structures such as caches?